tests/conftest.py

```diff
@@ -163,7 +172,8 @@ class DockerVirtualSwitch(object):
     FLEX_COUNTER_DB_ID = 5
     STATE_DB_ID = 6
 
-    def __init__(self, name=None, imgname=None, keeptb=False, fakeplatform=None):
+    def __init__(self, name=None, imgname=None, keeptb=False, fakeplatform=None, vct=None,
+                 newctnname=None, ctnmounts=None,vctdata=None):
```

Comment:
Can vctdata be part of vct ?

Reply:
sure, vctdata can be looked up by dvs through a call to vct.get_inband(ctnname)